### PR TITLE
Update voice-and-tone.mdx

### PR DESCRIPTION
**Pull request by CitizenFox**
Everything was reorganized and (almost) nothing deleted.

Reorganize the Voice and Tone page into meaningful groups of like content.

[Deploy Preview](https://deploy-preview-2114--keen-mayer-a86c8b.netlify.app/foundation/voice-and-tone)

#### What does this PR do?

There's too much content in the Voice and Tone for a single page so I want to do two things to reorganize it:

**This step**: Reorganize and divide content into five groups of like concepts.
- Content Intro
- Best Practices - High-level/conceptual/gray-area guidance for writing clear content
- Guidelines - Concrete rules for quick reference
- Implementation - Some writing-process best practices for developers and designers
- Case Study - A series of real-world examples for creating better content.
    
This is a starting point for meaningfully breaking up the Voice and Tone page. But this doesn't necessarily represent the destination. For instance, I think the case study is very helpful and there may be a clean way to merge it with our best practices to show best practices in action.
    
**Proposed Next Step**: Divide the content into separate sub-pages pages.
There's something to be said for having content all in one place but I believe it's more approachable (and so digestible) when it's divided into pages of well-defined concepts. If the division of content in step one proves helpful, I'll work with someone to divide the groups into Content section sub-pages.

#### Where should the reviewer start?

The Voice and Tone page.

#### What testing has been done on this PR?

In addition to the feature you are implementing, have you checked the following:

**General UX Checks**
- [x] Small, medium, and large screen sizes
- [x] Cross-browsers (FireFox, Chrome, and Safari)
- [x] Light & dark modes
- [x] All hyperlinks route properly

**Accessibility Checks**
- [x] Keyboard interactions
- [x] Screen reader experience
- [x] Run WAVE accessibility plugin (Chrome)

**Code Quality Checks**
- [x] Console is free of warnings and errors
- [x] Passes E2E commit checks
- [x] Visual snapshots are reasonable

### CODE REVIEW

**Comment by CitizenFox** [comment]
😑

**Comment by CitizenFox** [comment]
Can someone help me understand the 'unmet peer dependencies' in the deploy of the preview? I'm not sure how to resolve them.

**Comment by britt6612** [comment]
> Can someone help me understand the 'unmet peer dependencies' in the deploy of the preview? I'm not sure how to resolve them.

I can fix it for you!

**Comment by CitizenFox** [comment]
@britt6612 Thank you for the help!

**Review by britt6612 — APPROVED** [review]
Looks Good! The new flow looks fine to me

File: aries-site/src/pages/foundation/voice-and-tone.mdx [hunk 1]
```diff
@@ -22,24 +22,12 @@ We exercise empathy in our voice through situational tones that speak to each co
 - Fact-based
 - Plain English
 
-## Point of View
+# Best Practices
```
Comment (taysea) [inline aries-site/src/pages/foundation/voice-and-tone.mdx:25]:
```suggestion
## Best Practices
```

File: aries-site/src/pages/foundation/voice-and-tone.mdx [hunk 2]
```diff
@@ -68,6 +56,92 @@ The following are some high-level guidelines for constructing good messaging:
...
+
+## Quoting
+
+In messaging that contains user-specified content, use double-quotes around this content if it is not rendered as a hyperlink. If user-content is rendered as a hyperlink, the hyperlink styling is sufficient to offset the content from other text in the message.
+
+Why is this important? User-entered names can contain spaces, and if they are not quoted or rendered as hyperlinks, they can hinder comprehension of a message.
+
+Here is a correct example: _the uplink set "{0}" has a problem_. Compare this to an incorrect example: _the uplink set {0} has a problem_. If the uplink set is named "deleted", this will read as _the uplink set deleted has a problem_.
+
+# Guidelines
```
Comment (taysea) [inline aries-site/src/pages/foundation/voice-and-tone.mdx:141]:
```suggestion
## Guidelines
```

File: aries-site/src/pages/foundation/voice-and-tone.mdx [hunk 3]
```diff
@@ -179,11 +234,135 @@ various contexts. For example:
...
+hardcoded messages allows for a product's branding to be changed easily,
+confidently, and comprehensively.
+Lastly, consider formal and informal substitution values which can be used in
+various contexts. For example:
+
+- formal = HPE Ezmeral Container Platform
+- normal = Ezmeral Container Platform
+- short = Container Platform
+
+# Case Study
```
Comment (taysea) [inline aries-site/src/pages/foundation/voice-and-tone.mdx:361]:
```suggestion
## Case Study
```

File: aries-site/src/pages/foundation/voice-and-tone.mdx [hunk 4]
```diff
@@ -179,11 +234,135 @@ various contexts. For example:
...
 
 Consider a case with messaging related to a condition where a Kubernetes cluster is running low on available memory. Versions 1-7 highlight bad practices and identify the respective problems within each version. Lastly, a [best practice](#best-practice) example is provided.
 
-### Version 1
+## Version 1
```
Comment (taysea) [inline aries-site/src/pages/foundation/voice-and-tone.mdx:365]:
```suggestion
### Version 1
```

File: aries-site/src/pages/foundation/voice-and-tone.mdx [hunk 5]
```diff
@@ -195,7 +374,7 @@ Problems:
 - Robotic (robots don't use words like "a", "the", "is")
 - Incorrect capitalization of memory (it is not a proper noun)
 
-### Version 2
+## Version 2
```
Comment (taysea) [inline aries-site/src/pages/foundation/voice-and-tone.mdx:377]:
```suggestion
### Version 2
```

File: aries-site/src/pages/foundation/voice-and-tone.mdx [hunk 6]
```diff
@@ -205,7 +384,7 @@ Problems:
 - Names are user-specified and can cause readability issues if not quoted or hyperlinks. See what happens if "ABC" is replaced with names like "the", "storage", "Joe's", "obsolete", or any random string containing spaces.
 - What does "out of memory" actually mean? We should be more specific.
 
-### Version 3
+## Version 3
```
Comment (taysea) [inline aries-site/src/pages/foundation/voice-and-tone.mdx:387]:
```suggestion
### Version 3
```

File: aries-site/src/pages/foundation/voice-and-tone.mdx [hunk 7]
```diff
@@ -229,7 +408,7 @@ Problems:
 
 - What is the user supposed to do about this? Never raise an issue without offering a solution.
 
-### Version 5
+## Version 5
```
Comment (taysea) [inline aries-site/src/pages/foundation/voice-and-tone.mdx:411]:
```suggestion
### Version 5
```

File: aries-site/src/pages/foundation/voice-and-tone.mdx [hunk 8]
```diff
@@ -245,7 +424,7 @@ Problems:
 
 - What is the needful? This is a saying not used in conversational American English and does not give the user specific guidance.
 
-### Version 6
+## Version 6
```
Comment (taysea) [inline aries-site/src/pages/foundation/voice-and-tone.mdx:427]:
```suggestion
### Version 6
```

File: aries-site/src/pages/foundation/voice-and-tone.mdx [hunk 9]
```diff
@@ -265,7 +444,7 @@ Problems:
 - Is this really a critical issue? Unused resources are often considered wasted resources. It is actually a good thing to be fully utilizing your hardware.
 - In an aaS world, additional capacity should be a simple add-on to an existing service. As such, this may not be a critical issue, but could be transformed to a lower priority marketing message.
 
-### Version 7
+## Version 7
```
Comment (taysea) [inline aries-site/src/pages/foundation/voice-and-tone.mdx:447]:
```suggestion
### Version 7
```

File: aries-site/src/pages/foundation/voice-and-tone.mdx [hunk 10]
```diff
@@ -285,7 +464,7 @@ Problems:
 
 - If possible, give the user a way to directly perform a mentioned action.
 
-### Best Practice
+## Best Practice
```
Comment (taysea) [inline aries-site/src/pages/foundation/voice-and-tone.mdx:467]:
```suggestion
### Best Practice
```

File: aries-site/src/pages/foundation/voice-and-tone.mdx [hunk 11]
```diff
@@ -179,11 +234,135 @@ various contexts. For example:
...
+| and if                                                                 | Split into two sentences                                                | Bad example: "Retry the operation and if the problem persists, contact your authorized support representative and provide a support dump". Good example: "Retry the operation. If the problem persists, contact your authorized support representative and provide a support dump". In general, each separate option in an error resolution should be either, its own sentence, or in a comma separated list. The last-resort action of contacting support must always be in its own sentence. |
+| re-order                                                         | reorder                                                           | Use this to describe the action of reorganizing items in a list, rather than re-order (placing another order).
+
+# Implementation
```
Comment (taysea) [inline aries-site/src/pages/foundation/voice-and-tone.mdx:305]:
```suggestion
## Implementation
```

**Review by taysea — COMMENTED** [review]
@CitizenFox I know that these will ultimately get split out into difference pages, but for now while they're on the same page we can only have one h1 (#). The subsequent sections need to start with an h2 (##) and their sub-topics as h3 (###) -- would you mind updating to reflect that? Otherwise, the PR looks good.

**Pull request merged**